GitUI/Editor/FileViewerInternal.cs

```diff
@@ -60,7 +60,10 @@ public FileViewerInternal()
                         TextEditor.ActiveTextAreaControl.TextArea.TextView.GetLogicalLine(e.Y)));
             };
 
-            HighlightingManager.Manager.DefaultHighlighting.SetColorFor("LineNumbers", new HighlightColor(Color.FromArgb(80, 0, 0, 0), Color.White, false, false));
```

Comment:
The background looks like a visual artifact in the theme I test with and I suggest changing this (maybe as in https://github.com/gitextensions/gitextensions/pull/6651#issuecomment-497133999) also if GE normal theme changes.

Reply:
Right. I changed lines to just use System Control and Text colors. Looks good on my black/gray/white themes.